funcs/aws.go

```diff
@@ -30,14 +30,18 @@ func AWSFuncs(f map[string]interface{}) {
 	f["ec2dynamic"] = AWSNS().EC2Dynamic
 	f["ec2tag"] = AWSNS().EC2Tag
 	f["ec2region"] = AWSNS().EC2Region
+	f["kmsEncrypt"] = AWSNS().KMSEncrypt
+	f["kmsDecrypt"] = AWSNS().KMSDecrypt
```

Comment:
can we leave the aliases out for now? I don't think `aws.KMSEncrypt`/`aws.KMSDecrypt` are too cumbersome 🙂